packages/@o3r/testing/src/core/elements/programmatic-checkbox-element.ts

```diff
@@ -0,0 +1,36 @@
+import { TranspilationPurposeOnlyError } from '../../errors/index';
+
+import { O3rElement } from '../element';
+import { CheckboxElementProfile } from './checkbox-element';
+
+/**
+ * Interface to describe the Checkbox elements that are used inside a fixture.
+ * As for ComponentFixtureProfile, this abstracts the testing framework that is used by choosing the right
+ * implementation at runtime.
```

Comment:
Maybe you should specify that this fixture is provided only for the case of "complex" checkbox label